PS8lite.Rmd

```diff
@@ -187,23 +187,38 @@ MiscVal + MoSold + YrSold" %>%
 Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
-
+#create random forest model (2 nodes, 100 trees)
+model_rf <- randomForest(
+  form = model_formula,
+  data = training, 
+  mtry = 2,
+  ntree = 100 
+)
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a code review if this were real life. Then you would then make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step here.